src/library_webgpu.js

```diff
@@ -635,7 +635,7 @@ var LibraryWebGPU = {
     VertexStepMode: [
       'vertex',
       'instance',
-      'vertex-buffer-not-used',
+      undefined,
```

Comment:
Given the comment below it would make sense if this didn't even exist in this array. We would have to add some support to Dawn's generator though. (e.g. make `"jsrepr": null` tell the generator to skip it when generating JS)

For now it's fine if it exists.

Reply:
Ack